tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_reducers_test.ts

```diff
@@ -253,71 +253,92 @@ describe('Debugger reducers', () => {
-        expect(alertsOfType[0]).toEqual(alert0);
-        expect(alertsOfType[1]).toEqual(alert1);
-        expect(Object.keys(nextState.alerts.executionIndices)).toEqual([
-          AlertType.INF_NAN_ALERT,
-        ]);
-        const executionIndices =
-          nextState.alerts.executionIndices[AlertType.INF_NAN_ALERT];
-        expect(executionIndices).toEqual([
-          firstAlertExecutionIndex,
-          firstAlertExecutionIndex + 1,
-        ]);
-        // Verify that the first alert is scrolled into view.
-        expect(nextState.executions.scrollBeginIndex).toEqual(
-          expectedScrollBegin
-        );
-      });
+      it(
+        'Updates alerts data and scrollBeginIndex and graph-exec focusIndex: ' +
+          'empty initial state',
+        () => {
+          const firstAlertExecutionIndex = 10;
+          const firstAlertGraphExecutionIndex = 100;
+          const state = createDebuggerState({
+            activeRunId: '__default_debugger_run__',
+            alerts: createAlertsState({
+              alertsLoaded: {
+                state: DataLoadState.LOADING,
+                lastLoadedTimeInMs: null,
+              },
+            }),
+            executions: createDebuggerExecutionsState({
+              displayCount,
+              scrollBeginIndex: 0,
+            }),
+          }); // `alerts` state is in an empty initial state.
+          const alert0 = createTestInfNanAlert({
+            op_type: 'RealDiv',
+            execution_index: firstAlertExecutionIndex,
+            graph_execution_trace_index: 100,
+          });
+          const alert1 = createTestInfNanAlert({
+            op_type: 'Log',
+            execution_index: firstAlertExecutionIndex + 1,
+            graph_execution_trace_index: firstAlertGraphExecutionIndex * 2,
+          });
+          const nextState = reducers(
+            state,
+            actions.alertsOfTypeLoaded({
+              numAlerts: 2,
+              alertsBreakdown: {
+                InfNanAlert: 2,
+              },
+              begin: 0,
+              end: 2,
+              alertType: 'InfNanAlert',
+              alerts: [alert0, alert1],
+            })
+          );
+          expect(nextState.alerts.alertsLoaded.state).toBe(
+            DataLoadState.LOADED
+          );
+          expect(
+            nextState.alerts.alertsLoaded.lastLoadedTimeInMs
+          ).toBeGreaterThan(0);
+          expect(nextState.alerts.numAlerts).toBe(2);
+          expect(nextState.alerts.alertsBreakdown).toEqual({
+            [AlertType.INF_NAN_ALERT]: 2,
+          });
+          expect(Object.keys(nextState.alerts.alerts)).toEqual([
+            AlertType.INF_NAN_ALERT,
+          ]);
+          const alertsOfType = nextState.alerts.alerts[AlertType.INF_NAN_ALERT];
+          expect(Object.keys(alertsOfType).length).toBe(2);
+          expect(alertsOfType[0]).toEqual(alert0);
+          expect(alertsOfType[1]).toEqual(alert1);
```

Comment:
This would read much better if you wrote:

```ts
expect(nextState.alerts.alerts[AlertType.INF_NAN_ALERT]).toEqual([
  alert0,
  alert1,
]);
```

Similarly, below can be rewritten as:
```ts
expect(nextState.alerts.executionIndices).toEqual({
  [AlertType.INF_NAN_ALERT]: [firstAlertExecutionIndex, firstAlertExecutionIndex + 1],
});
```